.travis.yml

```diff
@@ -67,6 +66,7 @@ script:
   - python test/MatcherTests.py
   - python test/InferenceTests.py
   # Run notebooks end-to-end
+  - mkdir examples/gene_tag_example/gt
```

Comment:
Probably not...that was an artifact of something else and I just took it out. The logs seem to indicate it's an apt-get issue for the JDK